lib/onc_certification_g10_test_kit/bulk_data_group_export.rb

```diff
@@ -73,7 +73,12 @@ class BulkDataGroupExport < Inferno::TestGroup
             resource.type == 'Group' &&
               resource.respond_to?(:operation) &&
               resource.operation&.find do |operation|
-                operation.definition&.match(%r{^http://hl7.org/fhir/uv/bulkdata/OperationDefinition/group-export(\|\S+)?})
+                name_match = ['export', 'group-export'].include?(operation.name)
```

Comment:
I don't understand why we are allowing `group-export` in the name.  If we are, it seems to me that there isn't great consensus on what the right behavior should be, and we should just stop trying to enforce anything here...